website/docs/guides/getting_started.html.markdown

```diff
@@ -136,76 +136,51 @@ New Relic alerts are great, but they're even better when combined with good noti
 
 ```hcl
 # Notification channel
-resource "newrelic_alert_channel" "alert_notification_email" {
-  name = "username@example.com"
-  type = "email"
-
-  config {
-    recipients              = "username@example.com"
-    include_json_attachment = "1"
+resource "newrelic_notification_destination" "sample_notification_destination" {
+  name = "Sample Notification Destination"
+  type = "EMAIL"
+  
+  property {
+    key   = "email"
+    value = "username@example.com"
```

Comment:
A note for @kidk: Based on your feedback, I did want to remove the `property` block, but it looks like both `notification_destination` and `notification_channel` need to have atleast one `property` block, failing which I see an error ` Error: Insufficient property blocks: At least 1 "property" blocks are required.`.

This is why the `property` block has been added in the new example.